### PR TITLE
CSVS-110 E2E tests for VM events handling were implemented

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

**Comment by MedvedevBW** [comment]
@placccebo

File: src/it/scala/com/bwsw/cloudstack/vault/server/e2e/EndToEndTestSuite.scala [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+  val eventManager = new EventManager(
+    consumer,
+    mapper,
+    controller,
+    eventManagerSettings
+  )
+
+  Future(eventManager.execute())
+
+  it should "handle vm creation/deletion" in {
```
Comment (placccebo) [inline src/it/scala/com/bwsw/cloudstack/vault/server/e2e/EndToEndTestSuite.scala]: replace "it" with a project name

File: src/it/scala/com/bwsw/cloudstack/vault/server/e2e/EndToEndTestSuite.scala [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+    assert(zooKeeperService.getNodeData(writeTokenPath).contains(tokenTuple.writeToken))
+
+    //tokens permissions test
+    vaultPermissionTest(IntegrationTestsSettings.vmSecretPath, vmId, tokenTuple)
+
+    //delete VM
+    val vmDeleteRequest = new VmDeleteRequest(vmId)
+    executor.executeRequest(vmDeleteRequest.request)
+
+    //wait VM deletion handling
```
Comment (placccebo) [inline src/it/scala/com/bwsw/cloudstack/vault/server/e2e/EndToEndTestSuite.scala]: wait for

**Review by placccebo — CHANGES_REQUESTED** [review]

File: src/it/scala/com/bwsw/cloudstack/vault/server/e2e/EndToEndTestSuite.scala (hunk 1 above)
Comment (MedvedevBW) [reply src/it/scala/com/bwsw/cloudstack/vault/server/e2e/EndToEndTestSuite.scala]: ok

File: src/it/scala/com/bwsw/cloudstack/vault/server/e2e/EndToEndTestSuite.scala (hunk 2 above)
Comment (MedvedevBW) [reply src/it/scala/com/bwsw/cloudstack/vault/server/e2e/EndToEndTestSuite.scala]: ok

**Review by placccebo — APPROVED** [review]

**Pull request merged**